packages/client/components/AnalyticsPage.tsx

```diff
@@ -57,17 +64,37 @@ if (__PRODUCTION__ && datadogClientToken && datadogApplicationId && datadogServi
   })
   datadogRum.startSessionReplayRecording()
 }
+
 // page titles are changed in child components via useDocumentTitle, which fires after this
 // we must guarantee that this runs after useDocumentTitle
 // we can't move this into useDocumentTitle since the pathname may change without chaging the title
 const TIME_TO_RENDER_TREE = 100
 
 const AnalyticsPage = () => {
-  const key = window.__ACTION__.segment
-  if (!key) return null // development use
-  /* eslint-disable */
+  const atmosphere = useAtmosphere()
+  const [viewerInfo, setViewerInfo] = useState<ViewerInfo | undefined>(undefined)
+  const fetchViewerInfo = async () => {
+    const analyticsQueryRes = await atmosphere.fetchQuery<AnalyticsPageQuery>(query)
+    setViewerInfo(analyticsQueryRes?.viewer)
+  }
+  useEffect(() => {
+    fetchViewerInfo()
+  }, [atmosphere.viewerId])
+  useEffect(() => {
+    setupLogRocket(viewerInfo)
+    identifyUserWithDatadog(viewerInfo)
+    identifyUserWithSegment(viewerInfo)
+    identifyUserWithSentry(viewerInfo)
```

Comment:
My main goal with this change was to reduce the number of places we were identifying users with third-parties. Now, whenever the `viewerInfo` changes (when it's set or removed with a log out), we have this single effect that calls into all the relevant `identify` methods.

Previously, this method contained a duplicated call to the server endpoint. This also cleaned up the `ErrorBoundary` component, by moving all user identification to this consolidated place. Now, when adding another third-party plugin that needs to identify the user, there's a single place to look.